documentation/tutorials/kw_environment_prompt.rst

```diff
@@ -0,0 +1,50 @@
+=======================================================================
```

Comment:
You don't need to create a new tutorial; just expand the `kw env` tutorial.

https://kworkflow.org/tutorials/kw-env.html